### PR TITLE
Build R 4.0

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -155,6 +156,7 @@ requirements:
     - {{ pin_compatible('bzip2') }}      # [not win]
     - {{ pin_compatible('icu') }}        # [not win]
     - sed                                # [linux]
+    - pcre2
```
Comment (xhochy) [inline recipe/meta.yaml:159]: `pcre2` is missing a `run_exports`

**Comment by xhochy** [comment]
Main issue is the new pcre2 dependency. We need a `run_exports` via https://github.com/conda-forge/pcre2-feedstock/pull/20, a global pinning and Windows suppport, and build on `aarch64`, `ppc64le`.

**Comment by bgruening** [comment]
@xhochy thanks a lot!

**Comment by bgruening** [comment]
@xhochy I guess my commit disturbed the linter. Could you please rebase?

**Comment by CJ-Wright** [comment]
The pcre2 pin is going out now.

**Comment by dpryan79** [comment]
We can just hit rerun on the CI then in a bit then :)

**Comment by CJ-Wright** [comment]
Yes, I think the migration will be small, maybe one package (since R doesn't depend on it, yet)

**Comment by CJ-Wright** [comment]
It seems that the windows failure is because `msys2` doesn't have pcre2. @mingwandroid what should be done here?

**Comment by mingwandroid** [comment]
I am not sure! Will bring it up with the anaconda team.

**Comment by CJ-Wright** [comment]
Thank you very much!
Is there some place where the `{{ native }}` (and msys) things are documented?

**Comment by xhochy** [comment]
I cannot build this locally on Windows because I get 
```
gcc -pipe -std=gnu99  -s  -o Rpwd.exe rpwd.o rcico.o
collect2.exe: fatal error: open temporary output file: Permission denied
```

Any ideas what this could be?

**Comment by dpryan79** [comment]
At least it's a different error message now :)

**Comment by xhochy** [comment]
Also tried building locally with an `m2w64-pcre2` I downloaded from msys2 but this gives the same error.

**Comment by xhochy** [comment]
Opened an issue about the MinGW-toolchain in general: https://github.com/conda-forge/conda-forge.github.io/issues/1044

**Pull request closed without merging**